### DISCUSSION TITLE
Cascader有跟element的多选功能一样吗

### DISCUSSION THREAD

**Discussion by limuen** [ant-design/ant-design#51697] [Q&A]
在antd里面Cascader启用搜索showSearch以后，例如搜索1，过滤出有1的以后，选中一个以后就直接给消失了，element的是失去焦点才会消失

**Comment by petercat-assistant** [comment]
在 Ant Design 的 Cascader 组件中，启用搜索功能 (`showSearch`) 后，确实会在选择一个选项后自动关闭下拉菜单。这与 Element UI 的行为不同，Element UI 的 Cascader 组件在选择后不会立即关闭，而是等到失去焦点时才关闭。

如果你希望在 Ant Design 中实现类似 Element UI 的行为，可以通过自定义组件的行为来实现。例如，你可以通过监听 `onChange` 事件并阻止默认行为，或者通过其他方式来控制下拉菜单的显示和隐藏。

如果需要更具体的实现方案或代码示例，请告诉我！我可以帮助你进一步探索如何实现这一功能。

如需进一步帮助，请在评论中描述您的问题并@petercat-assistant与我开始对话。
